SearchQnA/docker/xeon/README.md

```diff
@@ -107,7 +107,7 @@ curl http://${host_ip}:3002/v1/embeddings\
   -H 'Content-Type: application/json'
 
 # web retriever microservice
-your_embedding=$(python3 -c "import random; embedding = [random.uniform(-1, 1) for _ in range(768)]; print(embedding)")
+export your_embedding=$(python3 -c "import random; embedding = [random.uniform(-1, 1) for _ in range(768)]; print(embedding)")
```

Comment:
some are python3 and others are python
please make them the same

Reply:
Unified to `python3`